test/e2e-networking-library.sh

```diff
@@ -21,7 +21,7 @@ function install_istio() {
 
   if [[ -z "${NET_ISTIO_COMMIT}" ]]; then
     # TODO: Figure out the commit of net-istio.yaml from net-istio.yaml
-    readonly NET_ISTIO_COMMIT="6bbca066373b21689b5d90381c27920533809e82"
+    readonly NET_ISTIO_COMMIT="8102cd3d32f05be1c58260a9717d532a4a6d2f60"
```

Comment:
We still haven't automated this? 😅

Reply:
@JRBANCEL Do you have a plan to improve this as per your https://github.com/knative-sandbox/net-istio/pull/288 ?

Reply:
There is plan. It just needs to do be executed 😄